### PR TITLE
Add Danish translation

### PR DESCRIPTION
**Pull request by thomasrosted**
Added Danish translation of the UI.

### CODE REVIEW

File: src/locales/da.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        "numbered": "Samtale %d",
+        "options": "Samtalemuligheder",
+        "setting": "Samtaleindstillinger",
+        "add": "Tilføj ny samtale",
+        "new": "Ny samtale",
+        "name": "Samtalenavn",
+        "edit": "Samtaleredigering",
+        "multiuser": "Opret flerbruger chat",
+        "delete": "Slet samtale",
+        "delete_confirm": "Er du sikker på du ønsker at slette samtale?",
```
Comment (danchr) [inline src/locales/da.json]: Dette bør være bestemt ental; enten “slette **denne** samtale?” eller “slette samtale**n**?”

File: src/locales/da.json [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        "leave": "Forlad samtale",
```
Comment (danchr) [inline src/locales/da.json]: do.

File: src/locales/da.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        "leave_confirm": "Er du sikker på du ønsker at forlade samtale?",
```
Comment (danchr) [inline src/locales/da.json]: do.

File: src/locales/da.json [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        "search": "Fremsøg mennesker",
```
Comment (danchr) [inline src/locales/da.json]: Evt. “**Søg efter* mennesker”?

File: src/locales/da.json [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+    "notification": {
+        "one": "Notifikation",
+        "other": "Notifikationer"
+    },
+    "favorite": {
+        "title": {
+            "one": "Favorit",
+            "other": "Favoritter"
+        },
+        "star_it": "Stjernemarkér / fjern stjernemarkering"
```
Comment (danchr) [inline src/locales/da.json]: “Tilføj / fjern stjernemarkering”?

File: src/locales/da.json [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+            "notification": {
+                "title": "Pop-op notifikationer",
+                "show": "Vis notifikationer",
+                "toggle": "Skift notifikationer",
+                "message": "Vis besked i notifikationer",
+                "username": "Vis brugernavn i notifikationer",
+                "mute": "Slå lyd i notifikationer fra",
+                "avatar": "Vis brugeravatar i notifikationer",
+                "icon": "Vis YakYak ikon i notifikationer",
+                "custom_sound": "Brug YakYak brugerdefineret lyd til notifikationer"
```
Comment (danchr) [inline src/locales/da.json]: Jeg ville foreslå “Brug særlig YakYak-lyd til notifikationer” — som sådan er lyden ikke brugerdefineret.

**Comment by averissimo** [comment]
Great work!

**Comment by averissimo** [comment]
@danchr and @thomasrosted I've merged the original request.

Can you guys reach a consensus on @danchr suggestions?

If any changes are required, please create another pull request :)

**Comment by averissimo** [comment]
My bad!! 

I was just  looking for the commits before including it in the pre-release and noticed it :)

**Pull request merged**